docs/problem_formulation/problem_formulation_ocp_mex.tex

```diff
@@ -487,123 +579,86 @@ \subsection{Terminal Constraints}\label{sec:constraints:terminal}
-    \label{tab:model:options}
-    \begin{tabular}{cccc}
-        \toprule
-        String id & Data type & Description & Required\\
-        \midrule
-        \code{name} & string & model name, used for code generation, default: \str{ocp\_model} & \optional \\
-        \code{T} & double & end time & \mandatory \\
-        \code{sym\_x} & \casadi{} expr. & state vector $x$ in problem formulation in sec.~\ref{sec:problem} & \mandatory \\
-        \code{sym\_u} & \casadi{} expr. & control vector $u$ in problem formulation in sec.~\ref{sec:problem} & only in OCP \\
-        \code{sym\_xdot} & \casadi{} expr. & derivative of the state $\dot{x}$ in implicit dynamics eq.~\eqref{eq:dynamics} & if IRK is used \\
-        \code{sym\_z} & \casadi{} expr. & algebraic state $z$ in implicit dynamics eq.~\eqref{eq:dynamics} & \optional, only with IRK \\
-        \code{sym\_p} & \casadi{} expr. & parameters $p$ of the problem formulation in sec.~\ref{sec:problem} & \optional \\
-        \multicolumn{4}{c}{$\vdots$}\\
-        \multicolumn{4}{c}{Additionally, options from Tables \ref{tab:dynamics}, \ref{tab:cost:slack}, \ref{tab:cost:auto}, \ref{tab:cost:external}, \ref{tab:cost:linear_ls}, \ref{tab:cost:nonlinear_ls}, %\ref{tab:constr_type},
-        \ref{tab:constraints:simplesyntax}, \ref{tab:constraints:extendedsyntax}, \ref{tab:constraints:path} and \ref{tab:constraints:terminal}, apply here.}\\
-        \multicolumn{4}{c}{$\vdots$}\\
-        \bottomrule
-    \end{tabular}%
-\end{table}%
-%
+
+
 \section{Solver \& Options}\label{sec:solver}
-%
-An instance of the solver options class is created by using: \code{ocp\_opts = acados\_ocp\_opts()}.
-Together with the model these options are used when instancing the solver interface class: \code{ocp = acados\_ocp(ocp\_model, ocp\_opts)}.\\
-Tables \ref{tab:solver_options}, \ref{tab:solver_options:qp_solver_option} and \ref{tab:solver_options:regularize_method} show (almost) all available options.
-These options are set in \matlab{} via \code{ocp\_opts.set(<stringid>, <value>)}.
-% Some options are not (yet) implemented in the \code{.set(*)}-interface and can be accessed via a struct by using \code{ocp\_opts.opts\_struct.(<stringid>) = <value>}.
-Furthermore, the struct \code{ocp\_opts.opts\_struct} and \href{https://github.com/acados/acados/blob/master/interfaces/acados_matlab_octave/acados_ocp_opts.m}{\code{acados\_ocp\_opts.m}} can be used as a reference for what other fields are available.
-\\
-Note that some options of the solver can be modified after creation using the routine: \code{set(<stringid>, <value>)}.
+
+
+Tables \ref{tab:solver_options}, \ref{tab:solver_options:qp_solver_option} and \ref{tab:solver_options:regularize_method} show some of the available options.
+A full list of the available options is given on the \href{https://docs.acados.org/python_interface/index.html#acados_template.acados_ocp_options.AcadosOcpOptions}{docs page generated from the docstrings in~\python}.
```

Comment:
Maybe even highlight, that we recommend to look up the options there instead of in the PDF.
Mention MATLAB specific options separately: I think it is just `compile_interface` right now.
Mention options class `AcadosOcpOptions`

Reply:
done